pom.xml

```diff
@@ -58,6 +58,16 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
+            <dependency>
```

Comment:
Is this dependency required? AssertJ comes from the parent POM so that should be removed.

Reply:
Two different versions got pulled in, which made maven print a warning when building (since it's unclear which one was expected). These lines in dependencyManagement just make it explicit which one should be used.

Reply:
I understand that, but I would prefer to manage this in a single place (the parent pom).